dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task222208CreateVariantTable.java

```diff
@@ -0,0 +1,42 @@
+package com.dotmarketing.startup.runonce;
+
+
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.common.db.DotDatabaseMetaData;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.StartupTask;
+import java.sql.SQLException;
+
+public class Task222208CreateVariantTable implements StartupTask {
+
+    private final String POSTGRES_QUERY = "CREATE TABLE IF NOT EXISTS variant ("
```

Comment:
good work using `IF NOT EXISTS`. We need this type of logic on all of our startup tasks